week-3/D-array-filter/exercise.js

```diff
@@ -8,8 +8,15 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var filterarr = [];
 
+pairsByIndexRaw.filter(function(item) {
+  if (item !== null && item !== false && item !== "whoops" && item.length > 1) {
+    filterarr.push(item);
```

Comment:
This will work, but it's not what we really want to do here. Remember that the `.filter()` method will create a new array for you, based on whether you return `true` or `false` for each element in the `pairsByIndexRaw` array.